components/serializer.rst

```diff
@@ -1206,6 +1206,10 @@ Option                          Description
 ``save_options``                XML saving `options with libxml`_                  ``0``
 ``remove_empty_tags``           If set to true, removes all empty tags in the      ``false``
                                 generated XML
+``cdata_wrapping``              If set to false, will not wrap any value           ``true``
+                                containing one of the following characters (
+                                ``<``, ``>``, ``&``) in `a CDATA section`_ like
+                                following : ``<![CDATA[...]]>``
 ==============================  =================================================  ==========================
 
```

Comment:
Please add a `versionadded` directive, you can find some as example in other files

Reply:
Done